src/commands/create.ts

```diff
@@ -54,10 +55,18 @@ export default class Create extends Command {
         const folder = path.join(process.cwd(), info.nameSlug);
 
         cli.action.start(`Creating a Rocket.Chat App in ${ chalk.green(folder) }`);
-
+        const { flags }  = this.parse(Create);
         const fd = new FolderDetails(this);
         fd.setAppInfo(info);
         fd.setFolder(folder);
+        if (flags.icon) {
+            const imagePathIcon = path.resolve(flags.icon);
+            if (fd.doesFileExist(imagePathIcon)) {
+                fd.setImageIconPath(imagePathIcon);
+            } else {
+                this.warn(new Error('Invalid image icon path detected, set default image'));
```

Comment:
I think it should error out and not be successful when the path they provided exists, at least I think I would expect it to fail. 🤔